lib/rack/honeycomb/middleware.rb

```diff
@@ -2,8 +2,13 @@
 
 module Rack
   module Honeycomb
+    # Prefix for attaching arbitrary metadata to the `env`. Will be deleted from
+    # the `env` once it's pulled off of the `env` and onto a Honeycomb event.
+    ENV_PREFIX = "honeycomb."
 
     class Middleware
+      HEADER_REGEX = /^#{ Regexp.escape ENV_PREFIX }/
```

Comment:
Belated nitpick: `HEADER_REGEX` -> `ENV_REGEX` (just cowboyed it myself: 267737a)